.editorconfig

```diff
@@ -0,0 +1,11 @@
+# http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
```

Comment:
im not that tied to it because editorconfig will do the auto tabbing for me... just think its easier to read. ill roll with what the group wants.